### PR TITLE
fix: Detect computed properties in complexity analysis

### LINKED ISSUE #15: [Bug]: Computed Property is ignored
### Bug Description

Property inside struct, class or enum (computed property) is ignored by complexityCalculator.

example:

```swift
struct S {
    var v: String {
        let rand = Int.random(in: 0..<10)
        // return rand == 0 || rand > 5
        if rand == 0 {
            return true
        }
        if rand > 5 {
            return true
        } else {
            return false
        }
    }
}
```

### Steps to Reproduce

1. Run example code on debug site.
2. Conform that result is empty.

Image

### Expected Behavior

Computed property should be also processed. (cyclomatic: 3, cognitive: 3)

### Actual Behavior

Process is skipped.

### Swift Code Sample

```swift
struct S {
    var v: String {
        let rand = Int.random(in: 0..<10)
        // return rand == 0 || rand > 5
        if rand == 0 {
            return true
        }
        if rand > 5 {
            return true
        } else {
            return false
        }
    }
}
```

### Command Used

```bash

```

### Output/Error Messages

```shell

```

### Platform

Linux (Ubuntu)

### OS Version

jammy

### Swift Version

6.1

### swift-complexity Version

0.0.5

### Pre-submission Checks

- [x] I have searched existing issues to ensure this is not a duplicate
- [x] I have tested with the latest version of swift-complexity
- [x] I have provided all the required information above

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by fummicc1**
## Overview

Fix for Issue #15: Computed properties with implicit getters (shorthand syntax) were ignored by the complexity calculator.

## Type of Change

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] 🧪 Test improvements

## Related Issues

Closes #15

### Core Changes

- Add `visit(VariableDeclSyntax)` to FunctionDetector to detect shorthand computed properties
- Improve `visit(AccessorDeclSyntax)` to include property name in output (e.g., "get" → "foo.get")
- Add helper methods: `extractPropertyName`, `extractComputedPropertySignature`, `findParentPropertyName`

### Tests

- Add test fixture `computed_property.swift` with shorthand getter, explicit get/set, and didSet examples
- Add 3 unit tests for computed property detection and complexity calculation

### Documentation

- N/A (internal implementation change)

### Test Plan

- [x] Unit tests pass
- [x] Integration tests pass
- [x] Manual testing completed
- [x] Complexity analysis on self passes

### Test Commands

```bash
swift test
swift run SwiftComplexityCLI /tmp/issue15_test.swift
```

### Self-Analysis Required

- [x] Complexity analysis run on changed files
- [x] No functions exceed threshold (15)
- [x] New complex code is justified and documented

**Results**: All functions in FunctionDetector.swift have complexity ≤ 4 (max cyclomatic: 4, max cognitive: 4)

### Platform Testing

- [x] Changes tested on macOS (Apple Silicon)
- [ ] Changes tested on macOS (Intel)
- [ ] Changes tested on Linux (Ubuntu 22.04+)
- [x] Platform-specific code properly conditionally compiled

### Platform Considerations

- [x] No platform-specific dependencies added
- [x] Linux compatibility maintained
- [x] macOS-specific features properly isolated

## Performance Impact

- [x] No performance impact

## Breaking Changes

- [x] No breaking changes

**Note**: Accessor naming improved (e.g., "get" → "foo.get") - this is additive, not breaking.

### Code Quality

- [x] Code follows the project's style guidelines
- [x] Self-review of code completed
- [x] Code is self-documenting with clear variable/function names
- [x] Complex logic is commented where necessary

### Testing

- [x] Tests added for new functionality
- [x] Existing tests updated as needed
- [x] All tests pass locally
- [x] Edge cases considered and tested

### Documentation

- [x] Documentation updated (if needed)
- [x] README updated (if needed) - N/A
- [x] CLAUDE.md updated (if architecture changes) - N/A
- [x] API documentation updated (if applicable) - N/A

### Dependencies

- [x] No new dependencies added

### CI/CD

- [ ] CI pipeline passes (macOS & Linux)
- [x] No new linting warnings introduced
- [x] Code formatting validation passes (swift-format on macOS)

### CODE REVIEW

File: Sources/SwiftComplexityCore/Analysis/FunctionDetector.swift [hunk 1]
```diff
@@ -139,4 +178,36 @@ class FunctionDetector: SyntaxVisitor {
...
+
+    private func extractComputedPropertySignature(
+        from variable: VariableDeclSyntax,
+        binding: PatternBindingSyntax
+    ) -> String {
+        var signature = variable.bindingSpecifier.text + " "  // "var" or "let"
+        signature += binding.pattern.description
+        if let typeAnnotation = binding.typeAnnotation {
+            signature += typeAnnotation.description
+        }
```
Comment (fummicc1) [inline Sources/SwiftComplexityCore/Analysis/FunctionDetector.swift]: [nits]

We can write like the following way:

```swift
[variable.bindingSpecifier.text, binding.pattern.description].join(" ")
```

writing it inline improves readability.
